src/Resources/views/storefront/page/product-detail/buy-widget-form/fast-order-modal.html.twig

```diff
@@ -2,15 +2,39 @@
 <div class="modal fade" id="fastOrderModal" tabindex="-1" role="dialog" aria-labelledby="fastOrderModalLabel"
      aria-hidden="true">
     <div class="modal-dialog" role="document">
-        <form method="post" class="fast-order-form" action="{{ path('frontend.fast-order.json') }}" data-form-ajax-submit="true">
+        <!-- TODO: if there an error: button should be available to resend; now it is blocked -->
```

Comment:
![NreNWjCIITU](https://github.com/IlyaPotseluev/sw6_fast_order/assets/19282486/ca26fce5-ce59-4382-9783-1314a4076887)

Reply:
That is how does it looks like. But if there're will be an error - place order will not unblocked. 
I know that, but it will be solved later.